corehq/apps/registration/templates/registration/confirmation_error.html

```diff
@@ -0,0 +1,13 @@
+{% extends "registration/base.html" %}
+
+{% block title %}Error Confirming Account{% endblock title %}
+
+{% block step-title %}
+    {{ message_title }}{% if message_subtitle %} <small>{{ message_subtitle }}</small>{% endif %}
+{% endblock %}
+
+{% block registration-content %}
+    {% if message_body %}<p>{{ message_body }}</p>{% endif %}
+    <p>You may try <a href="{% url "registration_resend_domain_confirmation" %}">resending the confirmation email</a>.</p>
+    <p>In the meantime, please explore the <a href="/a/public/">CommCare HQ Example Project.</a></p>
```

Comment:
@amsagoff do we still send people here? i didn't even know this thing still existed